pkgs/tools/misc/mermaid-filter/default.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, buildPackages
+, chromium
+}:
+
+buildNpmPackage rec {
+  pname = "mermaid-filter";
+  version = "1.4.6";
+
+  src = fetchFromGitHub {
+    owner = "raghur";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-5MKiUeiqEeWicOIdqOJ22x3VswYKiK4RSxZRzJntO6M=";
+  };
+
+  npmDepsHash = "sha256-pnylo3dPgj7aD5czTWSV+uP5Cj8rVAsjZYoJ/WPRuuc=";
+
+  nativeBuildInputs = [ buildPackages.makeWrapper ];
```

```suggestion
  nativeBuildInputs = [ buildPackages.makeWrapper ];
```
```suggestion
  nativeBuildInputs = [
    makeWrapper
  ];
```
We can call `makeWrapper` directly.